cinder-{{cookiecutter.driver_name_lc}}/tests/bundles/bionic-{{cookiecutter.release}}.yaml

```diff
@@ -27,20 +28,15 @@ applications:
   keystone:
     charm: cs:~openstack-charmers-next/keystone
     num_units: 1
-    options:
-      openstack-origin: cloud:xenial-{{ cookiecutter.release }}
```

Comment:
I think this should stay, otherwise the keystone release won't match the filename of this bundle?

Reply:
hm, it didn't seem to cause any trouble in my deploy. I guess I can reinstate it, with `bionic-` as the prefix.

Reply:
Yes it won't cause troubles but it then tests the wrong thing ;) I mean for example you end up with a bundle named "bionic-stein" and it tests Queens instead.